dhall/dhall.cabal

```diff
@@ -26,6 +26,7 @@ Extra-Source-Files:
     benchmark/deep-nested-large-record/*.dhall
     benchmark/examples/*.dhall
     benchmark/examples/normalize/*.dhall
+    man/dhall.1
```

Comment:
What's the motivation for including this file in the source tarball? `cabal` can't install man pages, can it?

Reply:
Nope! Some programs include them so that they're installed, e.g. xmonad, idris, darcs, and hlint.

On my computer:

```
vanessa@vanessa-desktop ~/programming/haskell/done/paseto 🌸 fd '\.1$' ~/.cabal/ -t f
/home/vanessa/.cabal/store/ghc-7.10.3/xmonad-0.14-f619cef269effbd1c8c51269a993c3a27b74c356324d42eceec12baa82dc2110/share/man/xmonad.1
/home/vanessa/.cabal/store/ghc-8.6.5/cabal-install-3.0.0.0-c8bf602282522d86f6e46259811b75eeea2643085f3e18a16d9656a73a116b75/share/man/man1/cabal.1
/home/vanessa/.cabal/store/ghc-8.6.5/darcs-2.14.2-c1c30f5d69e6ed1f9b61247b18116e1b9a993145c2bd9f83e34ec58463b00894/share/man/man1/darcs.1
/home/vanessa/.cabal/store/ghc-8.6.5/idris-1.3.2-bed91be6222d39e5c19463d1c317000993e791d993755befd7913185e045133d/share/man/man1/idris.1
/home/vanessa/.cabal/store/ghc-8.8.2/hlint-2.2.11-e469a6946aa1aa2fac4ba57aee60a611700cc60ff2ea5dfa9157cd645be79546/share/hlint.1
```